README.md

```diff
@@ -49,31 +50,133 @@ Apache EventMesh has a vast amount of features to help users achieve their goals
 - Easy Function develop and framework integration.
 
 ## Roadmap
+
 Please go to the [roadmap](https://eventmesh.apache.org/docs/roadmap) to get the release history and new features of Apache EventMesh.
 
 ## Subprojects
+
 - [EventMesh-site](https://github.com/apache/eventmesh-site): Apache official website resources for EventMesh.
 - [EventMesh-workflow](https://github.com/apache/eventmesh-workflow): Serverless workflow runtime for event Orchestration on EventMesh.
 - [EventMesh-dashboard](https://github.com/apache/eventmesh-dashboard): Operation and maintenance console of EventMesh.
 - [EventMesh-catalog](https://github.com/apache/eventmesh-catalog): Catalog service for event schema management using AsyncAPI.
 - [EventMesh-go](https://github.com/apache/eventmesh-go): A go implementation for EventMesh runtime.
 
 ## Quick start
-Here are the guidelines:
 
-[Step 1: Deploy eventmesh-store](https://eventmesh.apache.org/docs/instruction/store)
+This section of the guide will show you the steps to deploy EventMesh from [Local](#run-eventmesh-runtime-locally), [Docker](#run-eventmesh-runtime-in-docker), [K8s](#run-eventmesh-runtime-in-kubernetes).  
+
+This section guide is just to help you quickly get started with EventMesh deployment, start EventMesh according to the default configuration, if you need more detailed EventMesh deployment steps, please visit the [EventMesh official document](https://eventmesh.apache.org/docs/next/introduction).
```

Comment:
Basically, it's okay. It would be better to remove `/next` from this URL because the Next version displays a banner:

![image](https://github.com/apache/eventmesh/assets/41445332/164a2b2c-da00-4794-bbba-9eed913c3c0f)

Besides, is this sentence not completely deleted? `is just to help you quickly get started with EventMesh deployment`

Reply:
@Alonexc

Reply:
got it.